.github/workflows/reset-db.yaml

```diff
@@ -25,6 +25,7 @@ on:
           - ab
           - bl
           - rjm
+          - dk
```

Comment:
did you manually add this? the script should auto add this to the top under staging

Reply:
Yes @abroddrick , I manually added most of these. The scripts had a stroke at the build step, due to either me having docker composed up already or being in the wrong directory. I had to run the rest of the steps manually and then edit the files by hand.